crates/primitives/src/key.rs

```diff
@@ -202,27 +202,27 @@ impl Add<u64> for Key {
     }
 }
 
-impl<'a> Add<u64> for &'a Key {
+impl Add<u64> for &Key {
```

Comment:
Funnily I was not aware that it was nowadays possible to leave away explicit lifetimes in impl blocks like these. Just checked with Rust playground. Good to know ... a bit baffled though.

Reply:
Remove it for better understanding
Especially students who don't know the concept of lifetimes